puppet/modules/web/manifests/vhost/deb.pp

```diff
@@ -35,5 +37,7 @@
       mode    => '0700',
       content => template('freight/deploy_debs.erb'),
     }
+
+    User <| title == 'freightstage' |> -> User <| title == $user |>
```

Comment:
An alternative to let Puppet figure this out itself is to manage the group explicitly in the receiver_setup. If you choose to go that route, you should set `gid` on the user resource as well to be explicit.